remote/src/main/resources/reference.conf

```diff
@@ -852,7 +852,7 @@ pekko {
         # collected, which is not as efficient as reusing buffers in the pool.
         large-buffer-pool-size = 32
 
-        # For enabling testing features, such as blackhole in pekko-remote-testkit.
+        # For enabling testing features, such as black-hole in pekko-remote-testkit.
```

Comment:
blackhole is correct, don't change this

Reply:
Yes, but I think the `black-hole` may clear that `blackhole`, Never mind, let we keep `blackhole`.